### PR TITLE
add hw01

### PR DESCRIPTION
**Pull request by rsuh-python**
I did my first hw! Pls review

### CODE REVIEW

File: hw01.ipynb [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bbdd572d-f43b-4e12-b32a-ff10c92f448c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# your code here\n",
+    "print('I wrote some bad code')"
```
Comment (fortvivlan) [inline hw01.ipynb:29]: BAD CODE

**Review by fortvivlan — CHANGES_REQUESTED** [review]
I request changes

**Comment by rsuh-python** [comment]
:(

**Review by fortvivlan — APPROVED** [review]
OKAY

**Comment by rsuh-python** [comment]
Yay!

**Pull request merged**